layouts/partials/tw/main.html

```diff
@@ -1,20 +1,6 @@
 {{ $params := partialCached "helper/page-params" .Page .RelPermalink }}
 {{ $hasImage := .Param "image-size" | ne "hidden" }}
 {{ $isWide := .Param "image-size" | eq "wide" }}
-{{ $isOneCol := (not $hasImage | or $isWide) }}
```

Comment:
The old page had this logic to figure out if the page should be one column. Hiding the main image still works, but the image-size = wide doesn't work anymore. I think we can delay doing this, but I do want it to be a todo item to bring it back because some articles really do need to drop the right rail.